Assets/Plugins/Editor/BuildTool/DllCompiler.cs

```diff
@@ -61,6 +61,10 @@ public class DllCompiler {
 			Data.SdkLevel = 2;
 			Data.OutFile = UpliftDLL;
 			Data.useUnsafe = true;
+			Data.Defines = new string[]
+			{
```

Comment:
`Data.Defines = EditorUserBuildSettings.activeScriptCompilationDefines` should work!

Reply:
![image](https://user-images.githubusercontent.com/24282/33440433-3cb1f72e-d5f0-11e7-8094-c706d09e0247.png)

Reply:
Nice!